aws_lambda_builders/actions.py

```diff
@@ -159,3 +162,26 @@ def execute(self):
             new_destination = os.path.join(self.dest_dir, name)
 
             shutil.move(dependencies_source, new_destination)
+
+
+class CleanUpAction(BaseAction):
+
+    NAME = "CleanUp"
+
+    DESCRIPTION = "Cleaning up the target folder"
+
+    PURPOSE = Purpose.CLEAN_UP
+
+    def __init__(self, target_dir):
+        self.target_dir = target_dir
+
+    def execute(self):
+        targets = os.listdir(self.target_dir)
+
+        for name in targets:
+            target_path = os.path.join(self.target_dir, name)
```

Comment:
Maybe a little bit debug statement here that we are cleaning up this file/folder